src/lib/deployment.ts

```diff
@@ -64,11 +65,35 @@ export const storeCode = async ({
     fee: new Fee(store.fee.gasLimit, store.fee.amount),
   });
 
-  const res = await lcd.tx.broadcast(storeCodeTx);
-  cli.action.stop();
+  const result = await lcd.tx.broadcastSync(storeCodeTx);
+  if (typeof result.code !== 'undefined') {
+    return cli.error(result.raw_log);
+  }
+
+  let res;
+  for (let i = 0; i <= 100; i++) {
```

Comment:
@octalmage - would recommend lowering the number of tries in this loop. If we can't get the txInfo by (perhaps 10/20 tries) it makes sense to stop and notify the end user.

Reply:
That’s fair! This is about a 30 second timeout currently, which does seem like a reasonable amount of time to retry. Maybe setting the sleep to 500ms, and only looping 50 times would be more appropriate?